tools/appveyor.psm1

```diff
@@ -435,7 +435,11 @@ function Invoke-AppveyorFinish
             Publish-NuGetFeed -OutputPath .\nuget-artifacts -VersionSuffix $preReleaseVersion
         }
 
-        $artifacts += (Get-ChildItem .\nuget-artifacts -ErrorAction SilentlyContinue | ForEach-Object {$_.FullName})
+        $nugetArtifacts = Get-ChildItem .\nuget-artifacts -ErrorAction SilentlyContinue
+        foreach($artifact in $nugetArtifacts)
+        {
+            $null = $artifacts.Add($artifact.FullName)
+        }
```

Comment:
How about `$artifacts.AddRange`? It's faster for adding a collection of objects and it doesn't return anything.